server/zanata-frontend/src/app/editor/components/MTSuggestionsButton/index.js

```diff
@@ -0,0 +1,24 @@
+import Button from 'antd/lib/button'
+import 'antd/lib/button/style/css'
+import Tooltip from 'antd/lib/tooltip'
+import 'antd/lib/tooltip/style/css'
+import React from 'react'
+
+class MTSuggestionsButton extends React.Component {
+
+  render () {
+    const text = (<a href='https://translate.google.com/'><img src='http://zanata.org/images/white-short.png'
```

Comment:
Could you please rename that image file to something like translated-by-google-white-short.png?

Also, `title` would be a more descriptive name.